benchmarks/benchmark.js

```diff
@@ -129,6 +129,8 @@ function printResults (results) {
       ]
     })
 
+  console.log(results)
```

Comment:
```suggestion
```

Reply:
This is needed for debugging this and showing that it's using one sample anyway